### PR TITLE
Ruby upgrade

### PR DESCRIPTION
**Pull request by wkdewey**

### CODE REVIEW

File: config/public.yml [hunk 1]
```diff
@@ -8,8 +8,9 @@ default:
   tei_html_xsl: scripts/overrides/tei_to_html.xsl
 
 development:
+  api_version: "120"
```
Comment (nichgray) [inline config/public.yml:11]: Should this be api_version: "2.0"?

File: config/public.yml [hunk 2]
```diff
@@ -8,8 +8,9 @@ default:
...
   es_path: https://cdrhdev1.unl.edu/elastic
-  es_index: cdrhapidev-v1
+  es_index: cdrhapidev-v2
   site_url: https://cdrhdev1.unl.edu/bandhistory_rails
```
Comment (nichgray) [inline config/public.yml:14]: Remove private hostnames from public config file.

File: scripts/overrides/csv_to_es_image.rb [hunk 3]
```diff
@@ -29,12 +29,14 @@ def date_display
...
 
-  def publisher
-    @row["publisher"] || @row["publisher/repository"]
+  def citation
+    {
+      "publisher" => @row["publisher"] || @row["publisher/repository"]
+    }
   end
 
   def rights_holder
```
Comment (nichgray) [inline scripts/overrides/csv_to_es_image.rb:42]: source field will need to be updated as follows:   
```
def has_source
    {
      "title" => source_combine(@row)
    }
  end
```

File: config/public.yml (hunk 1 above)
Comment (nichgray) [reply config/public.yml:11]: Change has been made in https://github.com/CDRH/data_bandhistory/pull/4.

File: config/public.yml (hunk 2 above)
Comment (nichgray) [reply config/public.yml:14]: Change has been made in https://github.com/CDRH/data_bandhistory/pull/4.

**Review by techgique — APPROVED** [review]
Looks good to me, but would like Will to double-check Nikki's changes :+1:

**Comment by wkdewey** [comment]
When I generate HTML, it doesn't show up. See https://cdrhrailsdev.unl.edu/bandhistory/item/band.nsp.0320. Something is wrong with one of the paths in config.

**Review by wkdewey — COMMENTED** [review]
I approve these changes

**Comment by wkdewey** [comment]
> When I generate HTML, it doesn't show up. See https://cdrhrailsdev.unl.edu/bandhistory/item/band.nsp.0320. Something is wrong with one of the paths in config.

Created an issue for this (https://github.com/CDRH/bandhistory_rails/issues/35). If I view the pages at the intended routes the HTML shows up.

**Pull request merged**